logicrunner/logicrunner_unit_test.go

```diff
@@ -185,27 +185,13 @@ func (suite *LogicRunnerTestSuite) TestHandleAdditionalCallFromPreviousExecutor(
 	}{
 		{
 			name:                           "Happy path",
-			expectedClarifyPendingStateCtr: 1,
 			expectedStartQueueProcessorCtr: 1,
 		},
-		{
-			name:                           "ClarifyPendingState failed",
-			clarifyPendingStateResult:      fmt.Errorf("ClarifyPendingState failed"),
-			expectedClarifyPendingStateCtr: 1,
-		},
 		{
 			name:                           "StartQueueProcessorIfNeeded failed",
 			startQueueProcessorResult:      fmt.Errorf("StartQueueProcessorIfNeeded failed"),
-			expectedClarifyPendingStateCtr: 1,
 			expectedStartQueueProcessorCtr: 1,
 		},
-		{
-			name:                           "Both procedures fail",
-			clarifyPendingStateResult:      fmt.Errorf("ClarifyPendingState failed"),
-			startQueueProcessorResult:      fmt.Errorf("StartQueueProcessorIfNeeded failed"),
-			expectedClarifyPendingStateCtr: 1,
-			expectedStartQueueProcessorCtr: 0,
-		},
```

Comment:
Test should be fixed, not removed. Or rewritten according to the changes.

Reply:
Oops never mind. I though it's another test.

Reply:
please return tests back